migrations/migrations/20181025144923-create-user.js

```diff
@@ -27,14 +27,6 @@ module.exports = {
         isAlphanumeric: true,
         type: Sequelize.STRING
       },
-      role: {
```

Comment:
you cant delete things in migrations, create a new one deleting what you need

Reply:
the changes in the migration where done in  the last PR, as this branch is behind master changes are not seen